test/testchunkingng.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ *    This software is in the public domain, furnished "as is", without technical
+ *    support, and with no warranty, express or implied, as to its usefulness for
+ *    any purpose.
+ *
+ */
+
+#include <QtTest>
+#include "syncenginetestutils.h"
+#include <syncengine.h>
+
+using namespace OCC;
+
+class TestChunkingNG : public QObject
```

Comment:
I see the tmp folder being polluted with folders named ChunkingNG-* - is there some cleanup missing?

```sh
root@ci:/tmp/ChunkingNgTest-AQJojC/A# ls -lh
total 287M
-rw-r--r-- 1 jenkins jenkins 287M Nov 11 02:34 a0
-rw-r--r-- 1 jenkins jenkins    4 Nov  4 02:35 a1
-rw-r--r-- 1 jenkins jenkins    4 Nov  4 02:35 a2